src/shadowbox/model/errors.ts

```diff
@@ -53,3 +53,9 @@ export class AccessKeyConflict extends OutlineError {
     super(`Access key "${accessKeyId}" conflict`);
   }
 }
+
+export class PasswordConflict extends OutlineError {
+  constructor() {
+    super('Some access key have same password');
+  }
```

Comment:
I think something like this will be more helpful to the user:

```suggestion
  constructor(accessKey AccessKey) {
    super(`Access key ${accessKey} has the same password. Please specify a unique password for each access key');
  }
```

Reply:
I'm sorry, I meant `AccessKeyId`, like we do in `AccessKeyNotFound`:
```suggestion
  constructor(accessKeyId?: AccessKeyId) {
    super(`Access key ${accessKeyId} has the same password. Please specify a unique password for each access key');
  }
```